enterprise/cypher/compatibility-spec-suite/src/test/resources/blacklists/compatibility-31.txt

```diff
@@ -10,3 +10,37 @@ In-query call to procedure with explicit arguments that drops all result fields
 // procedure call bugs
 Standalone call to procedure with argument of type INTEGER accepts value of type FLOAT
 In-query call to procedure with argument of type INTEGER accepts value of type FLOAT
+
+// TCK side effects are incorrectly specified -- need update
```

Comment:
Not testing side effects in 2.3 is not as serious as not testing side effects in 3.1.